styles.css

```diff
@@ -89,11 +102,386 @@
+  color: #f55800;
+}
+
+.about-section {
+  background: #1c1a19;
+  padding: 100px 25px 0;
+}
+
+.about-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 42px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-text {
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+  color: #f4f5f7;
+}
+
+.about-button {
+  background: #ff6b00;
+  padding: 12px;
+  font-size: 15px;
+  margin-top: 20px;
+}
+
+.about-button:enabled {
+  background: #f55800;
+  color: #fff;
+  border: none;
+}
+
+.about-button:disabled {
+  background: #e5e5e5;
+  border: none;
+  color: #5b5957;
+}
+
+.about-button:hover {
+  border: 1px solid #f55800;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.about {
+  background-image: url("./images/about\ image\ .png");
+  background-repeat: no-repeat;
+  background-position: right 0% bottom 0%;
+  background-size: 120px;
+  height: 70vh;
+}
+
+.language {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+}
+
+.language-list {
+  margin: 10px auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 43px;
+  text-transform: capitalize;
+}
+
+.img {
+  margin-right: 8px;
+}
+
+.intro-text {
+  width: 250px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 23px;
+  line-height: 39px;
+  text-align: center;
+  color: #172b4d;
+  margin-left: 5rem;
+  margin-top: 30px;
+}
+
+.line {
+  width: 100%;
+}
+
+.get-button {
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  width: 131px;
+  padding: 12px;
+  font-size: 15px;
+  margin: 10px 1px;
+}
+
+.get-button:hover {
+  border: 1px solid #f55800;
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.get-button:disabled {
+  background: #e5e5e5;
+  border: none;
+  color: #5b5957;
+}
+
+.form {
+  font-family: sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 58px 0;
+}
+
+#fname,
+#email {
+  border-bottom: 1px solid #dbdbd7;
+  border-right-width: 0;
+  border-top-width: 0;
+  border-left-width: 0;
+  padding: 15px 0 15px 15px;
+  margin-top: 12px;
+  width: 327px;
+}
+
+.textarea {
+  width: 327px;
+  height: 114px;
+  background-color: #fbf8f7;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  padding-top: 12px;
+  font-family: 'inter', sans-serif;
+  padding-left: 12px;
+  margin-top: 20px;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 20px;
+  max-width: 200px;
+  margin: 70px 0 110px;
+  margin-left: 120px;
+  justify-items: center;
+  padding-right: 30px;
+}
+
+.social-logos {
+  color: #505f79;
+  width: 15px;
+  height: 15px;
+  margin-top: 10px;
+}
+
+.footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #f55800;
+  color: #fff;
+  text-align: center;
+  padding: 30px 20px;
```

Comment:
- [ ] [Optional] As I mentioned above about the footer section Please remove the position fixed property to match the given design.